v3/rmi_kb/equator/equator.json

```diff
@@ -0,0 +1,30 @@
+{
+	"name": "EQUATOR",
+	"vendorId": "0xB16B",
+	"productId": "0xE0A1",
+	"menus": ["qmk_rgb_matrix"],
+	"matrix": {"rows": 5, "cols": 16},
+	"layouts": {
+		"keymap": [
+      [{"y":0.08,"x":0.62,"c":"#777777"},"0,0",{"c":"#cccccc"},"0,1",{"x":9.76},"0,11",{"x":0},"0,12",{"c":"#aaaaaa","w":2},"0,13\n\n\n0,0",{"x":0},"0,15"],
+      [{"x":0.49,"w":1.5},"1,0",{"c":"#cccccc"},"1,1",{"x":9.01},"1,11","1,12","1,13",{"w":1.5},"1,14",{"c":"#aaaaaa"},"1,15"],
+      [{"x":0.35,"w":1.75},"2,0",{"c":"#cccccc"},"2,1",{"x":9.31},"2,11","2,12",{"c":"#777777","w":2.25},"2,14",{"c":"#aaaaaa"},"2,15"],
+      [{"x":0.2,"w":2.25},"3,0",{"c":"#cccccc"},"3,2",{"x":8.62},"3,11","3,12",{"c":"#aaaaaa","w":1.75},"3,13",{"c":"#777777"},"3,14",{"c":"#aaaaaa"},"3,15"],
+      [{"x":0.2,"w":1.25},"4,0",{"w":1.25},"4,1",{"x":9.62,"w":1.25},"4,11",{"w":1.25},"4,12",{"c":"#777777"},"4,13","4,14","4,15"],
+      [{"y":0.25,"x":14.25,"c":"#cccccc"},"0,13\n\n\n0,1",{"c":"#aaaaaa"},"0,14\n\n\n0,1"],
+      [{"r":8,"rx":2.75,"c":"#cccccc"},"0,2","0,3","0,4","0,5","0,6"],
+      [{"x":0.5},"1,2","1,3","1,4","1,5"],
+      [{"x":0.75},"2,2","2,3","2,4","2,5"],
+      [{"x":1.25},"3,3","3,4","3,5","3,6"],
+      [{"x":1.25,"c":"#aaaaaa","w":1.25},"4,3",{"c":"#cccccc","w":2.75},"4,5"],
+      [{"r":-8,"rx":12.75,"x":-4.5},"0,7","0,8","0,9","0,10"],
+      [{"x":-5},"1,7","1,8","1,9","1,10"],
+      [{"x":-4.75},"2,7","2,8","2,9","2,10"],
+      [{"x":-5.25},"3,7","3,8","3,9","3,10"],
+      [{"x":-5.25,"w":2.25},"4,8",{"c":"#aaaaaa","w":1.25},"4,9"]
+    ],
+    "labels": [
+      ["Backspace", "Full", "Split"]
+    ]
+	}
+}
```

Comment:
Some changes I made to align with the rest of the repo:

- added the keycode line for the lighting, since lighting is enabled
- changed the split backspace form a dropdown menu to a toggle for split
- applied overall code formatting

```suggestion
{
    "name": "EQUATOR",
    "vendorId": "0xB16B",
    "productId": "0xE0A1",
    "keycodes": [
        "qmk_lighting"
    ],
    "menus": [
        "qmk_rgb_matrix"
    ],
    "matrix": {
        "rows": 5,
        "cols": 16
    },
    "layouts": {
        "labels": [
            "Split Backspace"
        ],
        "keymap": [
            [
                {
                    "y": 0.08,
                    "x": 0.62,
                    "c": "#777777"
                },
                "0,0",
                {
                    "c": "#cccccc"
                },
                "0,1",
                {
                    "x": 9.76
                },
                "0,11",
                {
                    "x": 0
                },
                "0,12",
                {
                    "c": "#aaaaaa",
                    "w": 2
                },
                "0,13\n\n\n0,0",
                {
                    "x": 0
                },
                "0,15"
            ],
            [
                {
                    "x": 0.49,
                    "w": 1.5
                },
                "1,0",
                {
                    "c": "#cccccc"
                },
                "1,1",
                {
                    "x": 9.01
                },
                "1,11",
                "1,12",
                "1,13",
                {
                    "w": 1.5
                },
                "1,14",
                {
                    "c": "#aaaaaa"
                },
                "1,15"
            ],
            [
                {
                    "x": 0.35,
                    "w": 1.75
                },
                "2,0",
                {
                    "c": "#cccccc"
                },
                "2,1",
                {
                    "x": 9.31
                },
                "2,11",
                "2,12",
                {
                    "c": "#777777",
                    "w": 2.25
                },
                "2,14",
                {
                    "c": "#aaaaaa"
                },
                "2,15"
            ],
            [
                {
                    "x": 0.2,
                    "w": 2.25
                },
                "3,0",
                {
                    "c": "#cccccc"
                },
                "3,2",
                {
                    "x": 8.62
                },
                "3,11",
                "3,12",
                {
                    "c": "#aaaaaa",
                    "w": 1.75
                },
                "3,13",
                {
                    "c": "#777777"
                },
                "3,14",
                {
                    "c": "#aaaaaa"
                },
                "3,15"
            ],
            [
                {
                    "x": 0.2,
                    "w": 1.25
                },
                "4,0",
                {
                    "w": 1.25
                },
                "4,1",
                {
                    "x": 9.62,
                    "w": 1.25
                },
                "4,11",
                {
                    "w": 1.25
                },
                "4,12",
                {
                    "c": "#777777"
                },
                "4,13",
                "4,14",
                "4,15"
            ],
            [
                {
                    "y": 0.25,
                    "x": 14.25,
                    "c": "#cccccc"
                },
                "0,13\n\n\n0,1",
                {
                    "c": "#aaaaaa"
                },
                "0,14\n\n\n0,1"
            ],
            [
                {
                    "r": 8,
                    "rx": 2.75,
                    "c": "#cccccc"
                },
                "0,2",
                "0,3",
                "0,4",
                "0,5",
                "0,6"
            ],
            [
                {
                    "x": 0.5
                },
                "1,2",
                "1,3",
                "1,4",
                "1,5"
            ],
            [
                {
                    "x": 0.75
                },
                "2,2",
                "2,3",
                "2,4",
                "2,5"
            ],
            [
                {
                    "x": 1.25
                },
                "3,3",
                "3,4",
                "3,5",
                "3,6"
            ],
            [
                {
                    "x": 1.25,
                    "c": "#aaaaaa",
                    "w": 1.25
                },
                "4,3",
                {
                    "c": "#cccccc",
                    "w": 2.75
                },
                "4,5"
            ],
            [
                {
                    "r": -8,
                    "rx": 12.75,
                    "x": -4.5
                },
                "0,7",
                "0,8",
                "0,9",
                "0,10"
            ],
            [
                {
                    "x": -5
                },
                "1,7",
                "1,8",
                "1,9",
                "1,10"
            ],
            [
                {
                    "x": -4.75
                },
                "2,7",
                "2,8",
                "2,9",
                "2,10"
            ],
            [
                {
                    "x": -5.25
                },
                "3,7",
                "3,8",
                "3,9",
                "3,10"
            ],
            [
                {
                    "x": -5.25,
                    "w": 2.25
                },
                "4,8",
                {
                    "c": "#aaaaaa",
                    "w": 1.25
                },
                "4,9"
            ]
        ]
    }
}
```